nodes/sample_host.json

```diff
@@ -40,9 +40,10 @@
       "redirect_domain_names": ["<domain name>", "<domain name>", "<...>"],
       "ruby_version": "2.1.0",
       "ssl_info": {
-        "key": "<ssl key>",
-        "crt": "<ssl crt>"
-        },
+        "enabled": true,
+        "certificate": "The cert file, optional",
```

Comment:
Maybe communicate more clearly that it should be a path for a file (and not the content):

`"certificate": "path to the cert file, optional",
"certificate": "path to the key file"`